.rustfmt.toml

```diff
@@ -0,0 +1,13 @@
+use_small_heuristics = "Default"
+force_explicit_abi = true
+hard_tabs = false
+max_width = 100
+merge_derives = true
```

Comment:
`true` is the default
https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#merge_derives

Reply:
I think I would prefer this one to be `false`. It's easy to manually merge the derives, but there are occasions where it's nice to keep them separate for clarity or debugging purposes.

Reply:
Fair point.  I'll change it.